test/generateInput.py

```diff
@@ -77,7 +77,7 @@ def generate_blobs_csv():
 def generate_images_csv(multiplier):
 
     multiplier = multiplier // 2
-    path    = "input/images/"
+    path    = "images/"
```

Comment:
I liked having a separate "input" directory where the csv was and the the images were below, as were blobs.

Reply:
It's still going to end up there. The problem was that the path was relative to the cwd of the running code. With adb, you can run it from arbitrary folder, adjusting the CLI path.

This change mostly sets the image path relative to the csv path, and the DataCSV treats it as such.

Reply:
BTW, this should be applicable to anything having blobs. I will update the Video and Blobs too.